tests/integration/ceph_mgr_test.go

```diff
@@ -122,19 +119,61 @@ func (s *CephMgrSuite) execute(command []string) (error, string) {
 	return s.installer.Execute("ceph", orchCommand, s.namespace)
 }
 
+func (s *CephMgrSuite) enableRookOrchestrator() error {
+	logger.Info("Enabling Rook orchestrator module: <ceph mgr module enable rook --force>")
+	err, output := s.installer.Execute("ceph", []string{"mgr", "module", "enable", "rook", "--force"}, s.namespace)
+	logger.Infof("output: %s", output)
+	if err != nil {
+		logger.Infof("Not possible to enable rook orchestrator module: %q", err)
+		return err
+	}
+	logger.Info("Setting orchestrator backend to Rook .... <ceph orch set backend rook>")
+	err, output = s.execute([]string{"set", "backend", "rook"})
+	logger.Infof("output: %s", output)
+	if err != nil {
+		logger.Infof("Not possible to set rook as backend orchestrator module: %q", err)
+	}
+	return err
+}
+
 func (s *CephMgrSuite) waitForOrchestrationModule() {
 	var err error
+	var orchStatus map[string]string
+
+	err = s.enableRookOrchestrator()
+	if err != nil {
+		logger.Error("First attempt: Error trying to set Rook orchestrator module")
+	}
+
 	for timeout := 0; timeout < 30; timeout++ {
+		logger.Info("Waiting for rook orchestrator module enabled and ready ...")
 		err, output := s.execute([]string{"status"})
```

Comment:
Can you pass `-format json` and deserialize the json? Then no need to parse the output yourself.